lib/conncache.c

```diff
@@ -45,6 +49,26 @@
 
 #define HASHKEY_SIZE 128
 
+static void conncache_discard_conn(struct conncache *connc,
+                                   struct Curl_easy *last_data,
+                                   struct connectdata *conn,
+                                   bool aborted);
+static void connc_disconnect(struct Curl_easy *data,
+                             struct connectdata *conn,
+                             struct conncache *connc,
+                             bool do_shutdown);
+static void connc_run_conn_shutdown(struct Curl_easy *data,
+                                    struct connectdata *conn,
+                                    bool *done);
+static void connc_run_conn_shutdown_handler(struct Curl_easy *data,
+                                            struct connectdata *conn);
+static CURLcode conncache_update_shutdown_ev(struct Curl_multi *multi,
+                                             struct Curl_easy *data,
+                                             struct connectdata *conn);
+#ifdef DEBUGBUILD
+static void conncache_shutdown_all(struct conncache *connc, int timeout_ms);
```

Comment:
Both `connc_` and `conncache_` are used as prefixes for connection cache functions. Is the former one for the pool of closing connections? It's not obvious to me based on the name.

Reply:
Yeah, the static functions should all start with `connc_` for unity. Will change.

Reply:
all static functions now start with `connc_`.